config/default/manager_auth_proxy_patch.yaml

```
@@ -4,30 +4,53 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: controller-manager
-  namespace: node-feature-discovery-operator
+  namespace: node-feature-discovery
```

Argh, sorry about the back-and-forth. But should we actually use the `node-feature-discovery-operator` namespace as default

```suggestion
  namespace: node-feature-discovery-operator
```

Ditto to other operator-related objects

---

yup, I was thinking in some NFD stuff while making the change so forgot it was operator 😅